### DISCUSSION TITLE
License question: Can I use TailwindUI for open source projects?

### DISCUSSION THREAD

**Discussion by corysimmons** [tailwindlabs/tailwindcss#8810] [Help] [answered]
I have a Personal License. I'd like to use the Syntax for [Bun](https://bun.sh)'s docs.

@adamwathan

1. Do you have any qualms with that? It'd be good marketing for TailwindUI.
2. If the the docs are completely OSS, then the template would be viewable by everyone. I suppose a few maintainers could be given copies of the `src` and we could somehow isolate the open code to being just a `docs` dir or something. Does anyone on the team have experience doing something like that and could offer some insight?

**Comment by adamwathan** [comment, marked as answer]
@corysimmons Hey! For things like this shoot us an email at support@tailwindui.com, that's the best place to discuss licensing questions 👍

For anyone else reading — this answer is intended specifically for @corysimmons, if you have similar questions for your own project please reach out to us via support to get an answer or permission for your situation, even if it seems the same as this one.

Quick answer — we try to be crazy permissive and generous with our licensing compared to most template products out there but still have to draw the line somewhere, or we risk not being able to sustain working on Tailwind or anything at all.

So with that in mind I'm cool with it with a few requirements:

1. Whoever actually owns the project/repo is the person who owns a Tailwind UI license. We can't let people scaffold out open-source Tailwind UI-based projects for other people unfortunately, that's against the spirit of the license.

2. The documentation repo can't have an OSS license, as that would effectively give people permission to use it for their own projects without purchasing. I'd ask that in the README you include a "License" section that's explicit about this, with this content:

    > This project is open for contribution but the source code itself uses a commercial template and is therefore not licensed under any open-source license. Forking this project as a base for your own projects is not permitted under the license of the original template.

    If you want to include a LICENSE file as well, I'd ask that it just includes that statement to make it clear to anyone looking for it that no license is actually provided.

Alternatively you can keep the actual documentation site shell closed source and open-source your actual content files in a separate repo like you've mentioned, that would be totally cool of course.

Hope that sounds fair — like I said we do try to be extremely generous with our licensing but need to be a little careful too if we want to be able to keep working on these projects ❤️

**Reply by corysimmons** [reply]
I understand. I don't think I can say I'm going to maintain those docs forever and be the only person responsible for running builds, so I don't think it'd be a good fit.

Any chance of offering that Syntax one for free with a link to TailwindUI in the sidebar or something? 😁 Might be good for marketing purposes.

**Reply by YourTechBud** [reply]
I was going through the licensing FAQs and it seems like tailwind ui can be used in open source projects now. Is that accurate? If so, maybe we should update this answer to reflect that.
